docs/contributing/Compiler Test Plan.md

```diff
@@ -37,9 +37,15 @@ This document provides guidance for thinking about language interactions and tes
  
 # Type and members
 - Access modifiers (public, protected, internal, protected internal, private protected, private), static, ref
-- type declarations (class, record class/struct with or without positional members, struct, interface, type parameter)
+- type declarations
+  - class with or without primary constructor
+  - record class/struct with or without positional members
+  - struct with or without primary constructor
+  - interface
+  - type parameter
 - file-local types
 - methods
+  - Primary constructors
```

Comment:
Minor comment: do we really need to call out primary ctors here? We don't call out regular ctors?

Reply:
> Minor comment: do we really need to call out primary ctors here? We don't call out regular ctors?
 
I can remove this if you feel strongly. To me, it doesn't look like this hurts anything.

Reply:
I don't feel strongly. That's why I approved. I was mainly curious about the motivations for listing in multiple places.